realm/realm-annotations-processor/src/main/java/io/realm/processor/RealmProxyClassGenerator.java

```diff
@@ -1006,8 +1006,8 @@ private void emitInsertOrUpdateMethod(JavaWriter writer) throws IOException {
             } else if (Utils.isRealmList(field)) {
                 writer
                         .emitEmptyLine()
-                        .emitStatement("long nativeLinkViewPtr = Table.nativeGetLinkView(tableNativePtr, columnInfo.%sIndex, rowIndex)", fieldName)
-                        .emitStatement("LinkView.nativeClear(nativeLinkViewPtr)")
+                        .emitStatement("long nativeLinkViewPtr%1$s = Table.nativeGetLinkView(tableNativePtr, columnInfo.%1$sIndex, rowIndex)", fieldName)
```

Comment:
How about `%1$sNativeLinkViewPtr` instead of `nativeLinkViewPtr%1$s`.
`%1$sNativeLinkViewPtr` is more natural as a variable name `columnRealmListNativeLinkViewPtr` vs `nativeLinkViewPtrcolumnRealmList`.